projects/tod_simulator/scripts/get_quick_eval_stats.py

```diff
@@ -0,0 +1,208 @@
+
+            base = ""
+            multitask = ""
+            if not os.path.isfile(base_path + "run.sh"):
+                continue
+            with open(base_path + "run.sh") as run_file:
+                for line in run_file:
+                    if "zoo:bart" in line:
+                        base = "BartOnlyNoApi"
+                    if "8a7" in line:
+                        base = "GsgdOnlyNoApiLvl0"
+                    if "8ca" in line:
+                        base = "GoodNoApiLvl0"
+                    if "1fe" in line:
+                        base = "AllNoApi0"
+                    if "e28" in line:
+                        base = "AllHasApi0"
+                    if "840" in line:
+                        base = "GsgdOnlyNoApiRL1"
+                    if "1eb" in line:
+                        base = "AllNoApiRL1"
+                    if "076" in line:
+                        base = "AllYesApiRL1"
+                    if "79a" in line:
+                        base = "SysGsgdOnly"
+                    if "c31" in line:
+                        base = "SysGood"
+                    if "ba9" in line:
+                        base = "SysAll"
+
+            with open(base_path + "model.opt") as opt_file:
+                print(base_path)
+                opt = json.load(opt_file)
+                lr = opt["learningrate"]
+                multitask = opt.get("multitask_weights", "")
+                yes_api = opt.get("api_schemas", False)
+            if multitask != "":
+                multitask = "_" + "".join([str(x) for x in multitask])
+
+            orig_path = "/".join(os.path.abspath(base_path).split("/")[-3:])
+
+            ppl = eval_data["report"].get("ppl", "")
+            token_em = eval_data["report"].get("token_em", "")
+            jga = eval_data["report"].get("jga", "")
+            jga_n = eval_data["report"].get("jga_noempty", "")
+            jga_e = eval_data["report"].get("jga_empty", "")
+
+            if yes_api:  # hack cause I'm annoyed at things being in wrong alpha order
+                yes_api = "True"
+            else:
+                yes_api = "false"
+
+            maybe_nshot = ""
+            root = base_path
+            eval_file = glob.glob(f"{root}/*mm_eval*.json")
+            if len(eval_file) > 0 and (
+                "percent" in eval_file[0] or "nshot" in eval_file[0]
+            ):
+                maybe_nshot = eval_file[0][len(root + "mm_eval_") : -len(".json")]
+
+            base = (
+                str(len(opt.get("task").split(",")))
+                + str(yes_api)
+                + "-APIS__"
+                + maybe_nshot
+                + base
+                + "_"
+                + str(lr)
+                + multitask
+            )
+            metrics = [
+                orig_path,
+                base,
+                str(ppl),
+                str(token_em),
+                str(jga),
+                str(jga_n),
+                str(jga_e),
+            ]
```

Comment:
eee fstring?

Reply:
Yeah I can change it to that lol.

Did it this way initially cause having everything on separate lines was slightly more readable when I was initially going through this code...